systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAbstractST.java

```diff
@@ -101,6 +105,7 @@ private void createSecretsForDeployments() {
         SecretUtils.createSecret(MIRROR_MAKER_OAUTH_SECRET, OAUTH_KEY, "a2Fma2EtbWlycm9yLW1ha2VyLXNlY3JldA==");
         SecretUtils.createSecret(MIRROR_MAKER_2_OAUTH_SECRET, OAUTH_KEY, "a2Fma2EtbWlycm9yLW1ha2VyLTItc2VjcmV0");
         SecretUtils.createSecret(BRIDGE_OAUTH_SECRET, OAUTH_KEY, "a2Fma2EtYnJpZGdlLXNlY3JldA==");
+        SecretUtils.createSecret(OAUTH_CLIENT_AUDIENCE_SECRET, OAUTH_KEY, "a2Fma2EtYXVkaWVuY2Utc2VjcmV0");
     }
```

Comment:
It might be worthy to have string with real value and do base64 from it during test. WDYT? For sure not in this PR

Reply:
I totally agree. It would be more visible. 
Or, I would suggest to move encoding into `SecretUtils.createSecret()` method, where it is created on openshift.